src/data/real/ennreal.lean

```diff
@@ -698,12 +666,12 @@ begin
   calc ↑(a * b) < ↑(a' * b') :
     coe_lt_coe.2 (mul_lt_mul' aa'.le bb' (zero_le _) ((zero_le a).trans_lt aa'))
   ... = ↑a' * ↑b' : coe_mul
-  ... ≤ c * d : mul_le_mul a'c.le b'd.le
+  ... ≤ c * d : mul_le_mul' a'c.le b'd.le
 end
 
-lemma mul_left_mono : monotone ((*) a) := λ b c, mul_le_mul le_rfl
+lemma mul_left_mono : monotone ((*) a) := λ b c, mul_le_mul' le_rfl
```

Comment:
Could this lemma and the next one instead be generalized to the case where `mul_le_mul'` holds? They don't look specific to `ennreal`.

Reply:
I would prefer to avoid modifications to ported files unless necessary.

Reply:
Fair enough. Maybe add a todo?